tests/test_socket_server.py

```diff
@@ -32,24 +32,20 @@
     format="%(asctime)s:%(levelname)s:%(name)s:%(message)s", level=logging.DEBUG
 )
 
-# Standard timeout in seconds.
 TIMEOUT = 5
+"""Standard timeout in seconds."""
```

Comment:
I suggest you revert this change.

Or have you found a way to make sphinx actually use the doc strings after module constants? If you have, please tell me how to do it.